mods/cnc/chrome/settings-display.yaml

```diff
@@ -182,6 +182,19 @@ Container@DISPLAY_PANEL:
 									Height: 20
 									Font: Regular
 									Text: Show Game Event Notifications
+				Container@ROW:
+					Width: PARENT_RIGHT - 24
+					Height: 20
+					Children:
+						Container@MUTE_REPLAY_CHAT_CHECKBOX_CONTAINER:
+							X: 10
+							Width: PARENT_RIGHT / 2 - 10
+							Children:
+								Checkbox@MUTE_REPLAY_CHAT_CHECKBOX:
+									Width: PARENT_RIGHT
+									Height: 20
+									Font: Regular
+									Text: Mute Replay Chat
```

Comment:
`Hide Chat in Replays` might be a bit clearer. (Muting could also mean only removing the sounds.) Opinions?

Reply:
I think hide works better